src/vm/eeconfig.cpp

```diff
@@ -789,10 +792,18 @@ HRESULT EEConfig::sync()
         g_fEnableARM = TRUE;
     }
 
-    int forceGCconcurrent = CLRConfig::GetConfigValue(CLRConfig::UNSUPPORTED_gcConcurrent);
-    if ((forceGCconcurrent > 0) || (forceGCconcurrent == -1 && g_IGCconcurrent))
+    bool gcConcurrentWasForced = false;
+#ifdef FEATURE_CORECLR
```

Comment:
We can't simply ifdef it out entirely because a bit further down, we need to know whether concurrent GC was forced. If it wasn't, we disable concurrent GC if GCStress is enabled.

However, doing the second option does mean that we won't treat the presence of `"System.GC.Concurrent": true` as the user forcing concurrent GC (in other words, we may turn it off even if they specify it).

If we're okay with that, I can undo the change in clrconfigvalues.h and here, and change `InitializeStartupFlags` to fallback to 'true' if "System.GC.Concurrent" isn't set.